### PR TITLE
Move about page to sidebar

### PR DESCRIPTION
**Pull request by t-pyrope**

### CODE REVIEW

**Comment by t-pyrope** [comment]
screencapture-localhost-3000-2021-07-14-11_03_25

screencapture-localhost-3000-2021-07-14-11_04_05

File: src/modules/about/index.js [hunk 1]
```diff
@@ -3,15 +3,21 @@ import Module from '../../abc/Module';
...
-			path: '/about',
+			path: "/about",
 			exact: true,
-			name: 'About',
-			component: AboutScreen
+			name: "About",
+			component: AboutScreen,
 		});
+
+		// app.Navigation.addItem({
```
Comment (Pe5h4) [inline src/modules/about/index.js:17]: @t-pyrope Why is the navigation commented?

Comment (Pe5h4) [reply src/modules/about/index.js:17]: Here is the proper place to add a navigation, not in `asab-config` module

File: src/modules/asab-config/index.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react';
 import Module from 'asab-webui/abc/Module';
 import ConfigContainer from "./ConfigContainer";
 import MicroservicesContainer from "./MicroservicesContainer";
-
+import AboutScreen from '../about/AboutScreen';
```
Comment (Pe5h4) [inline src/modules/asab-config/index.js:5]: @t-pyrope This is wrong, you dont want to import `About` screen to the `asab-config` module, please remove your implemented code from this `src/modules/asab-config/index.js` file

**Review by Pe5h4 — CHANGES_REQUESTED** [review]
Also, I miss the update in `CHANGELOG` file

**Comment by t-pyrope** [comment]
screencapture-chrome-extension-fdpohaocaechififmbbbbbknoalclacl-capture-html-2021-07-15-08_34_35

File: src/modules/asab-config/index.js [hunk 3]
```diff
@@ -39,9 +38,8 @@ export default class ConfigModule extends Module {
 					name: "Microservices",
 					url: "/config/microservices",
 					icon: 'cil-list',
-				}
+				},
```
Comment (Pe5h4) [inline src/modules/asab-config/index.js:41]: @t-pyrope It is not neccessary to have a comma separator here

**Review by Pe5h4 — APPROVED** [review]

**Review by ateska — APPROVED** [review]

**Pull request merged**